src/signals/incident/components/IncidentClassification/index.js

```diff
@@ -0,0 +1,32 @@
+import React, { useEffect } from 'react';
+import { useFetch } from 'hooks';
+import configuration from 'shared/services/configuration/configuration';
+
+import { useHistory, useParams } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { setClassification } from 'signals/incident/containers/IncidentContainer/actions';
+
+const IncidentClassification = () => {
+  const history = useHistory();
+  const { category, subcategory } = useParams();
+  const { get, data, error } = useFetch();
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    get(`${configuration.CATEGORIES_ENDPOINT}${category}/sub_categories/${subcategory}`);
+  }, [category, subcategory, get]);
+
+  useEffect(() => {
+    if (!data && !error) return;
```

Comment:
What if there is a `500` or `404` response from the API? Can you add a test for that?

Reply:
tests added

Reply:
Why not explicitly write:
```
    if (data) dispatch(setClassification({ category, subcategory }));
    if (data || error) history.push('/');
```

Reply:
Good suggestions, thanks!